tests/selection.py

```diff
@@ -97,22 +136,41 @@ def test_others_orthogonal(self):
                         EPSILON,
                     )
 
+    def test_features(self):
+        """
+        This test checks CUR returns the standard set of features
+        """
+        X, Y = load_boston(return_X_y=True)
+        model = self.model(X=X, mixing=0.5, Y=Y)
+        for i in range(1, len(self.idx) + 1):
+            with self.subTest(i=i):
+                model.select(n=i)
+                self.assertEqual(model.idx[-1], self.idx[i - 1])
+
 
 class SampleFPSTest(unittest.TestCase, PCovSelectionTest):
-    def setup(self):
+    def setUp(self):
         self.model = SampleFPS
 
+    def test_supplied_indices(self):
+        """
+        This test checks FPS will accept pre-defined indices.
+        """
+        X, Y = load_boston(return_X_y=True)
+        model = self.model(X=X, mixing=0.5, Y=Y, idxs=[488])
+        model.select(n=4)
+
 
 class SampleCURTest(unittest.TestCase, PCovSelectionTest):
-    def setup(self):
+    def setUp(self):
         self.model = SampleCUR
 
     def test_orthogonalize(self):
         """
         This test checks that the orthogonalization that occurs after each
         selection leads to a null sample row
         """
-        self.setup()
+        self.setUp()
```

Comment:
shouldn't this function be called automatically?

Reply:
Yes, it should, and was... fixing.